google/resource_container_node_pool.go

```diff
@@ -129,10 +180,70 @@ func resourceContainerNodePoolRead(d *schema.ResourceData, meta interface{}) err
 
 	d.Set("name", nodePool.Name)
 	d.Set("initial_node_count", nodePool.InitialNodeCount)
+	if nodePool.Autoscaling != nil && nodePool.Autoscaling.Enabled {
```

Comment:
Ah, that's a good point.

I went ahead and checked what the cloud console does- when you turn autoscaling off and then back on, you lose the min/max node count you had originally set. Based on that, I removed the enabled attribute so we can easily detect out-of-band changes (this way we don't need to differentiate between set to false and not set at all, although in retrospect maybe a `DiffSuppressFn` on the `autoscaling` attribute would work? not sure). Let me know what you think.